packages/google_identity_services_web/tool/run_tests.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Runs `dart test -p chrome` in the root of the google_identity_services_web package.
+//
+// Called from the custom-tests CI action.
+//
+// usage: dart run tool/run_tests.dart
+// (needs a `chrome` executable in $PATH, or a tweak to dart_test.yaml)
```

Comment:
Looks like you'll need to exclude this package from custom tests on non-Linux platforms in the CI config (unless you want to set up a version of install_chromium for other platforms).

Reply:
I've tweaked the CI so this behaves similarly to `cross_file` wrt the custom tests.